components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -2,172 +2,194 @@ $white:#ffffff;
 $black:#000000;
 $light-gray:#aaaaaa;
 $dark-gray:#2e2e2e;
-$gold:#c28e0e;
+$campusGold:#c28e0e;
```

Comment:
Like Jacob mentioned. The color variables are already defined, so you don't need to define your own here. just use `$color-campusGold` in your code below.

Reply:
Yep, the only colors not defined are $black, $light-gray, and $dark-gray. You can continue using those.